### ISSUE TITLE
Scheduler - Month View - Difference between dates of month

### ISSUE THREAD

**Issue by niyaza3** [telerik/kendo-react#402]
Hi,
We have integrated scheduler into our application. The issue that we face is there is no difference between previous and next month dates in month view. we need to style previous and next month dates with different css styles. can you provide more documentation on that please ?

**Comment by simonssspirit** [comment]
We are currently working on custom renders for the slots, which may prove helpful in this case.

@kspeyanski, please take a look at this use case.

**Comment by kspeyanski** [comment]
Yes, we are actively working on it, but for now i can not give an estimate when we will release it.

What we're aiming to achieve is to provide the following syntax for such customisations:
```jsx
const CustomDaySlot = (props) => {
return (<DaySlot
    {...props} 
    style={...props.style, { backgroundColor:  props.start.getMonth() !== props.date.getMonth() 
        ? {'light-gray' 
        :  undefined
      }  
  />)
}
```

I will update this issue once we've published a `dev` version with the implementation.

**Comment by niyaza3** [comment]
Thanks @kspeyanski  we will be waiting to hear from you soon.

**Comment by kspeyanski** [comment]
Hello @niyaza3 

Please excuse me for the delayed response.

The described behavior, together with more custom rendering properties have been published with version [`3.10.0`](https://www.telerik.com/kendo-react-ui/components/changelogs/ui-for-react/#toc-3100) of the `@progress/kendo-react-scheduler` package.

I've prepared the following stackblitz example, demonstrating the styling of a slot when outside the selected month:
https://stackblitz.com/edit/react-84wfnx

For more information about custom rendering inside the Scheduler, you can refer to the following articles:
- [Items Customisation](https://www.telerik.com/kendo-react-ui/components/scheduler/items/overview/)
- [Slots Customisation](https://www.telerik.com/kendo-react-ui/components/scheduler/slots/overview/)

Please let me know if you need further assistance, and once again, apologies for the delayed response.
